apsuite/commisslib/meas_fofb_sysid.py

```diff
@@ -278,12 +278,12 @@ def get_levels_bpms_from_svd(self, lvl0, lvl1, ampmin=0):
         us = u[:, singval]
         us /= _np.abs(us).max()
```

Comment:
I'm wondering if strictly speaking we shouldn't remove this division here... The vectors are normalized, so in order to keep the relationship between them and the control effort, we should keep them as is, instead of scaling using the biggest value, since that's kind of arbitrary (some modes can be more spread out than others, for example).

@fernandohds564 @guilhermerc what do you think? (adding you two since you aren't reviewers)

Reply:
I think at the time this was implemented you guys wanted to respect the limits defined by the levels, but I agree that it is better to remove this line to make things easier.